app/src/main/java/org/mozilla/fenix/library/history/HistoryView.kt

```diff
@@ -131,6 +135,13 @@ class HistoryView(
             )
             recentlyClosedNav.isVisible = !userHasHistory
         }
+        with(binding.syncedHistoryNavEmpty) {
+            val syncedHistoryVisible = FeatureFlags.syncedHistoryFeature && !userHasHistory
+            syncedHistoryNav.setOnClickListener {
+                interactor.onSyncedHistoryClicked()
+            }
+            syncedHistoryNav.isVisible = syncedHistoryVisible
+        }
```

Comment:
```suggestion

        with(binding.syncedHistoryNavEmpty) {
            val syncedHistoryVisible = FeatureFlags.syncedHistoryFeature && !userHasHistory
            syncedHistoryNav.setOnClickListener {
                interactor.onSyncedHistoryClicked()
            }
            syncedHistoryNav.isVisible = syncedHistoryVisible
        }

```

Add a new line before and after these kinda with/if/when blocks to the code some separation.

Reply:
fixed